### PR TITLE
Fix several small issues

### PR DESCRIPTION
**Pull request by jessefjxm**
1. Use {type = "logistic-container"} to filter out all these kind of containers. Support bob's logistics now.
2. Move main update section into a tick event that runs every 30 ticks. So it won't freeze the inventory when too much biters died in one tick.
3. Please ignore some localization I made. I just feel lazy to move them into the locale files :)

### CODE REVIEW

File: alien-module/control.lua [hunk 1]
```diff
@@ -36,7 +36,7 @@ function update_gui()
 		if player.gui.top.killcount == nil then
 			player.gui.top.add{type="label", name="killcount", caption="TEST"}
 		end
-		player.gui.top.killcount.caption = "Hypermodule level: " .. roundModuleLevel(3) .. " Killcount: " .. global.killcount
+		player.gui.top.killcount.caption = "异星超级插件等级: " .. roundModuleLevel(3) .. " 消灭异虫数量: " .. global.killcount
```
Comment (renoth) [inline alien-module/control.lua]: Thanks for the translation but i think the english text would be preferable here

File: alien-module/control.lua [hunk 2]
```diff
@@ -14,7 +14,7 @@ end)
 function init_gui()
 	for _, player in pairs(game.players) do
         player.gui.top.add{type="label", name="killcount", caption="Killcount"}
-		player.gui.top.killcount.caption = "Killcount: " .. global.killcount
+		player.gui.top.killcount.caption = "消灭异虫数量: " .. global.killcount
```
Comment (renoth) [inline alien-module/control.lua]: Thanks for the translation but i think the english text would be preferable here

File: alien-module/control.lua [hunk 3]
```diff
@@ -173,7 +170,7 @@ script.on_event(defines.events.on_entity_died, function(event)
 				end
 			end
 			
-			pp("Alien Hyper Modules upgraded to level: " .. global.modulelevel)
+			pp("异星超级插件提升至: " .. global.modulelevel .. "级")
```
Comment (renoth) [inline alien-module/control.lua]: Thanks for the translation but i think the english text would be preferable here

**Comment by jessefjxm** [comment]
Sure! I just feel like if we can localized those key words showing in gui would be perfect. I'm not sure if we can call localized strings in any where inside the mod luas.

**Comment by renoth** [comment]
Localized String API looks like this apparrently: 
http://lua-api.factorio.com/latest/Concepts.html#LocalisedString

**Comment by renoth** [comment]
Thank you very much for the PR, i commited the localization string loading in another PR! Should work with Chinese now as well.

**Pull request merged**